sympy/functions/special/error_functions.py

```diff
@@ -1928,11 +1927,11 @@ def _eval_aseries(self, n, args0, x, logx):
         # Expansion at oo
         if point is S.Infinity:
             z = self.args[0]
-            p = [S.NegativeOne**k * factorial(2*k) / z**(2*k)
-                    for k in range(int((n - 1)/2))] + [Order(1/z**n, x)]
-            q = [S.NegativeOne**k * factorial(2*k + 1) / z**(2*k + 1)
-                    for k in range(int(n/2) - 1)] + [Order(1/z**n, x)]
-            return pi/2 - (cos(z)/z)*Add(*p) - (sin(z)/z)*Add(*q)
+            p = [S.NegativeOne**k * factorial(2*k) / z**(2*k + 1)
+                    for k in range(int(n/2) + 1)] + [Order(1/z**n, x)]
```

Comment:
Rather than `int(n/2)` which uses floating point arithmetic it would be better to use `n // 2`.